src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -75,6 +77,9 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges {
   private childFieldArray: Array<any> = [];
   private childForms: any;
 
+  private _onRecordSelectedSubscription: Subscription;
```

Comment:
The resolution to this isn't quite what I meant. Instead of:

`this._appStateServiceSaveClickedSubscription = this.appStateService.saveClicked$.subscribe(() => {`
...
`this._subsription.add(this._appStateServiceSaveClickedSubscription);`

You can just add it directly and skip the variable:

`this._subscription.add(this.appStateService.saveClicked$.subscribe(() => {`

Reply:
This is a lot more succinct and it means you don't need to have these three variables defined at all